client/extensions/wp-super-cache/preload-tab.jsx

```diff
@@ -30,28 +30,37 @@ import { isPreloadingCache } from './state/cache/selectors';
  */
 const renderCachePreloadInterval = ( {
 	handleChange,
-	isRequesting,
-	isSaving,
+	isDisabled,
 	preload_interval,
 } ) => (
 	<FormTextInput
 		className="wp-super-cache__preload-interval"
-		disabled={ isRequesting || isSaving }
+		disabled={ isDisabled }
 		min="0"
 		name="preload_interval"
 		onChange={ handleChange( 'preload_interval' ) }
 		step="1"
 		type="number"
-		value={ preload_interval || '' } />
+		value={ preload_interval || '0' } />
 );
 
 class PreloadTab extends Component {
-	state = {
-		preloadRefresh: true,
+	componentWillReceiveProps( nextProps ) {
+		this.state = {
+			preloadRefresh: parseInt( nextProps.fields.preload_interval, 10 ) === 0 ? false : true
```

Comment:
Sure, adjusted in https://github.com/Automattic/wp-calypso/pull/14518/commits/5757b015ecaf4c715653b51871e3f0a083002824. But just curious, what clarity does that change provide? Is it simply because it's strange seeing `false` in the place usually reserved for a true expression?